sourcegraph/sourcegraph.proto

```diff
@@ -355,6 +355,8 @@ message ReposListBranchesOp {
 }
 
 message RepoListBranchesOptions {
+	bool include_commit = 4;
```

Comment:
I don't really know how these protobuf ids work in full detail. I should read up about it and have a better idea.

I just went with incrementing the highest id when adding new fields.